content/specific-skill-success-criteria/basic-data-analysis/_index.md

```diff
@@ -5,4 +5,151 @@ title: 'Assessment: Basic data analysis'
+Therefore the mean (middle) value is:  median = 350
+
+
+When we have an odd amount of numbers we need to divide the total amount of numbers by 2 and then
+round up to the nearest integer.  For instance:
+
+list_of_speed_data = [100, 200, 300, 400, 500, 600, 700]
+
+There are seven numbers in the list, 7/2 = 3.5, if we round up to the nearest integer we get 4.  
+Therefore, the 4th number in the list will be the median (middle) value:  median = 400
+```
+
+
+#### Mode (Most frequent value):
+```
+The mode is the value which occurs the most in a set.  So if you had a set [2, 2, 2, 3] then the
+mode would be 2 since 2 is the value which occurs most frequently in the set.
+```
+
+
+
+### Statistical concepts: Interquartile range (IQR)
+1st quartile:  Middle number between the smallest number and the median of the dataset
+
+2nd quartile:  Equal to the median of the dataset
+
+3rd quartile:  Middle number between the median and the largest value of the dataset
+
+
+
+### Statistical concepts: Standard deviation and Variance
+Standard deviation and variance are both determined by using the mean of the dataset.  Variance measures the average degree to which each number is different from the mean.  Standard deviation is a statistic that looks at how far from the mean a group of numbers is, by using the square root of the variance.  Standard deviation is similar to variance, which gives the measure of deviation whereas variance provides the value squared.
+
+```
+Formula for standard deviation:  square root of [(1/N)*Sigma i=1 to N of (x - u)^2 ]
+sample_data = [9, 2, 5, 4, 12, 7, 8, 11]
+
+Step 1: Calculate the Mean (average value) = (9+2+5+4+12+7+8+11)/8 = 7.25
+
+Step 2:  For each number in sample_date, subtract the mean from the number and square the result
+
+(9-7.25)^2 = 3.06
+(2-7.25)^2 = 27.56
+(5-7.25)^2 = 5.06
+(4-7.25)^2 = 10.56
+(12-7.25)^2 = 22.56
+(7-7.25)^2 = 0.06
+(8-7.25)^2 = 0.56
+(11-7.25)^2 = 14.06
+
+[3.06, 27.56, 5.06, 10.56, 22.56, 0.06, 0.56, 14.06]
+
+Step 3:  Calculate the mean of all the numbers squared you calculated in step 2.  It is important to notice
+         that once you have calculated step 3 you already have the Variance, the calculation done in step 4
+         will simply give you the standard deviation.
+
+(3.06+27.56+5.06+10.56+22.56+0.06+0.56+14.06)/8 = 10.435 (This is the variance)
+
+Step 4:  Take the square root of step 3 and you are done
+
+Square root of 10.435 = 3.23
+```
+
+
+
+### Statistical concepts: Correlation and p-value (Probability value)
+A correlation coefficient is represented in Python by the variable R, the value ranges between -1 and 1.  A correlation coefficient of -1 (Negative slope) or +1 (Positive slope) illustrates a perfect linear relationship between dependant and independent variables, a correlation coefficient of 0 indicates NO linear relationship between dependant and independent variables.
```

Comment:
I'm not sure that we should be explaining everything in depth here. This is more like assessment criteria.

Is this stuff not explained elsewhere in the syllabus? 

Can we do something smaller and more versatile. Eg: if the `p-value is equal to 0 then what does that tell us about correlation? How about if it equal to 1? And what is the lowest number that a p-value can take?` 

Learners should be able to answer that. If they can then that's a step in the right direction but it doesn't prove understanding. What other questions can we ask the learner so that they can prove that they actually understand?

Maybe we can ask then to work something out? 

Also I dont agree with your calculations here.

The chance of throwing a 1 on a 6 sided die is 1/6. The chance of throwing a 2 is also 1/6. Unless it's a weighted die. The chance of throwing a 6 is definately not 6/6 otherwise dice wouldn't work. 

And then you go on to say "So in English we would say you have a two in six chance of landing on a 6" but that doesn't relate to the previous statements you made.

I think this will just confuse people.

It might actually be good to get people to calculate these basic probabilities on thieir own. Eg:
If I flip a single coin then I have a 50% chance of getting a head. If I were to flip 2 coins what would be the chance of getting 2 heads? If I were to flip 3 coins then what are the chances of getting 3 heads? If I were to roll 2 6 sided dice,  what are the chances that the values would add up to 5? 

A lot of people struggle to reason about that sort of thing

Reply:
I took away some of the explanations and left some question marks where I feel a candidate should be able to answer the question.  I've also added a section of 10 questions.  I think it is ok to have some explanations with some gaps before the candidate gets to the 10 questions part, that way he/she kind of has a refresher